app/layouts/helpers.tsx

```diff
@@ -14,20 +14,24 @@ import { useScrollRestoration } from '~/hooks/use-scroll-restoration'
 import { SkipLinkTarget } from '~/ui/lib/SkipLink'
 import { classed } from '~/util/classed'
 
-export const PageContainer = classed.div`grid h-screen grid-cols-[14.25rem,1fr] grid-rows-[60px,1fr]`
+export const PageContainer = classed.div`h-full pt-[var(--navigation-height)] [overscroll-behavior-y:none]`
 
 export function ContentPane() {
   const ref = useRef<HTMLDivElement>(null)
   useScrollRestoration(ref)
   return (
-    <div ref={ref} className="flex flex-col overflow-auto" data-testid="scroll-container">
-      <div className="flex flex-grow flex-col pb-8">
+    <div
+      ref={ref}
+      className="flex flex-col lg+:ml-[var(--sidebar-width)] lg-:col-span-2"
+      id="content_pane"
+    >
+      <div className="flex flex-grow flex-col pb-8 md-:pb-16">
         <SkipLinkTarget />
         <main className="[&>*]:gutter">
           <Outlet />
         </main>
       </div>
-      <div className="sticky bottom-0 flex-shrink-0 justify-between overflow-hidden border-t bg-default border-secondary empty:border-t-0">
+      <div className="sticky bottom-0 z-popover flex-shrink-0 justify-between overflow-hidden border-t bg-default border-secondary empty:border-t-0">
```

Comment:
this might conflict with actual popovers

Reply:
<img width="573" alt="image" src="https://github.com/oxidecomputer/console/assets/4020798/58093497-35b6-4b8d-8284-8bee3a33a502">

<img width="270" alt="image" src="https://github.com/oxidecomputer/console/assets/4020798/1c428167-6c2a-43ad-bcdc-97364cf78571">

This seems to be fine because all of the popovers are in portals and therefore appears later in the DOM. I _could_ add a `z-bottomBar` or something that is `z-index: 5;` (popover is currently 10).